### PR TITLE
multimessage txs

### LINKED ISSUE #2149: Support multimessage transactions
**Is your feature request related to a problem? Please describe.**

Some transactions have multiple `msg` values, which we do not support yet as we're relying only on the first message. For eg:

```
{
  "height": "193168",
  "txhash": "7E129DFC4EC8887E50136E51252EA50F95FE3ECCEE1F56F722914975913E9106",
  "log": "[{\"msg_index\":\"0\",\"success\":true,\"log\":\"\"},{\"msg_index\":\"1\",\"success\":true,\"log\":\"\"}]",
  "gas_wanted": "200000",
  "gas_used": "96158",
  "tags": [
    {
      "key": "action",
      "value": "withdraw_delegation_reward"
    },
    {
      "key": "delegator",
      "value": "cosmos185n9jln2g4w79w39m0g85x2tns6uekqgzrhnap"
    },
    {
      "key": "source-validator",
      "value": "cosmosvaloper185n9jln2g4w79w39m0g85x2tns6uekqg8hrx3j"
    },
    {
      "key": "action",
      "value": "withdraw_validator_rewards_all"
    },
    {
      "key": "source-validator",
      "value": "cosmosvaloper185n9jln2g4w79w39m0g85x2tns6uekqg8hrx3j"
    }
  ],
  "tx": {
    "type": "auth/StdTx",
    "value": {
      "msg": [
        {
          "type": "cosmos-sdk/MsgWithdrawDelegationReward",
          "value": {
            "delegator_addr": "cosmos185n9jln2g4w79w39m0g85x2tns6uekqgzrhnap",
            "validator_addr": "cosmosvaloper185n9jln2g4w79w39m0g85x2tns6uekqg8hrx3j"
          }
        },
        {
          "type": "cosmos-sdk/MsgWithdrawValidatorCommission",
          "value": {
            "validator_addr": "cosmosvaloper185n9jln2g4w79w39m0g85x2tns6uekqg8hrx3j"
          }
        }
      ],
      "fee": {
        "amount": [
          {
            "denom": "photino",
            "amount": "10000"
          }
        ],
        "gas": "200000"
      },
      "signatures": [
        {
          "pub_key": {
            "type": "tendermint/PubKeySecp256k1",
            "value": "AgwqSG/R/k+kUtxO1tVqAMwGvuTDGslMGHdyr8gV7GY7"
          },
          "signature": "aa1hCv1TAEyAr/KYB3bHwX5t+drIKF6u0heeD+vzlUUKUez55w9gJBsf2MORBC5JtnH2ATh+wF3ieKudwH2FkQ=="
        }
      ],
      "memo": ""
    }
  }
}
```

**Describe the solution you'd like**

Support multiple message transactions on Voyager

**Describe alternatives you've considered**

a. Display both messages on the transaction (separated by a divider), or 
b. Display multiple transactions with the same tx height and metadata (Li transactions)

**Additional context**

[Mintscan](https://www.mintscan.io/txs/7E129DFC4EC8887E50136E51252EA50F95FE3ECCEE1F56F722914975913E9106) explorer uses the `a.` approach:

screen shot 2019-03-04 at 9 40 06 am

### PR DESCRIPTION
**Pull request by fedekunze**
Closes #2149 

**Description:**

Screen Shot 2019-03-23 at 5 18 18 PM

Thank you! 🚀

---

For contributor:

- [x] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/renderer/components/transactions/LiTransaction.vue [hunk 1]
```diff
@@ -44,7 +44,7 @@ export default {
       default: null // TODO: fails with required: true
     },
     block: {
-      type: [String, Number], // for convenience we allow strings and numbers
```
Comment (faboweb) [inline app/src/renderer/components/transactions/LiTransaction.vue]: nice

File: app/src/renderer/components/staking/TabMyDelegations.vue [hunk 2]
```diff
@@ -28,22 +28,29 @@
...
-              transaction.tx.value.fee.amount[0]"
-            :unbonding-time="
-              time.getUnbondingTime(
-                transaction,
-                delegation.unbondingDelegations
-              )
-            "
-            tx-type="cosmos-sdk/MsgUndelegate"
-          />
+          <div
```
Comment (faboweb) [inline app/src/renderer/components/staking/TabMyDelegations.vue]: how about using "LiAnyTransaction" to avoid the code duplication?

**Comment by faboweb** [comment]
Could we add a test that shows multi message transaction displaying?

**Comment by fedekunze** [comment]
@faboweb updated

**Pull request merged**